server/channels/api4/bot.go

```diff
@@ -128,7 +128,7 @@ func getBot(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if c.App.SessionHasPermissionTo(*c.AppContext.Session(), model.PermissionReadOthersBots) {
+	if c.App.SessionHasPermissionTo(*c.AppContext.Session(), model.PermissionReadOthersBots) { //nolint:revive
```

Comment:
The required `nolint` here, presumably because the block is empty, is unfortunate, as the "right" way to organize this conditional block is decidedly less clear. Is the pattern going forward to `nolint` these common things, or will we end up forcing developers to write their permissions checks more awkwardly?

Reply:
Personally I think that having to keep an empty block like that is more of a sign of a lazy function design.

One potential solution could be to create a local function wrapping the lines currently at the bottom of the method and call it when needed. Another option could be to use a `defer` function and check for the presence of `c.Err`. 

In cases where none of that works I am thinking the use of boolean variables can help a lot in making the logic digestible without resorting to this. But again, it's my very personal point of view.

Reply:
Your suggestions are definitely what I would expect in a code review from a human being. But it feels like encroaching into a gray zone if a bot wants to dictate how I write my if conditions. I am slightly leaning towards Jesse's thoughts to remove the if-condition checks. Let me know if you have strong opinions.